lib/sqlalchemy/sql/functions.py

```diff
@@ -1797,3 +1801,21 @@ class grouping_sets(GenericFunction[_T]):
     """
     _has_args = True
     inherit_cache = True
+
+
+class string_agg(GenericFunction[_T]):
```

Comment:
STRING_AGG is not a SQL standard function and here we are making it resolve to different kinds of function names.

so let's name it unambiguously something SQLAlchemy only, I propose:

``aggregate_strings(expr, delimeter)``


that way existing code calling `func.string_agg()` is not impacted

Reply:
good idea